frontend/src/components/OdhExploreCard.tsx

```diff
@@ -12,18 +12,29 @@ type OdhExploreCardProps = {
 };
 
 const OdhExploreCard: React.FC<OdhExploreCardProps> = ({ odhApp, isSelected, onSelect }) => {
+  const cardClasses = classNames('odh-card', { 'm-disabled': odhApp.spec.comingSoon });
   const badgeClasses = classNames('odh-card__partner-badge', {
     'm-warning': odhApp.spec.support === 'other',
   });
+
   return (
-    <Card isHoverable isSelectable isSelected={isSelected} className="odh-card" onClick={onSelect}>
+    <Card
+      isHoverable={!odhApp.spec.comingSoon}
+      isSelectable={!odhApp.spec.comingSoon}
+      isSelected={isSelected}
+      className={cardClasses}
+      onClick={onSelect}
```

Comment:
should you be able to click the card and have the drawer show up when it's in the "Coming soon" state?

Reply:
nope, thanks